cloudevent-broker/dashboard.tf

```diff
@@ -0,0 +1,88 @@
+module "sent-events" {
+  source = "../dashboard/tiles/xy"
+  title  = "Events Sent"
+  filter = [
+    "resource.type=\"pubsub_topic\"",
+    "metric.type=\"pubsub.googleapis.com/topic/send_request_count\"",
```

Comment:
We should filter this on the topic name prefix

Reply:
See my other comment, but I think you factored this out (with prefix!) and then forgot to clean this up?

Reply:
oh weird, no.

Reply:
The "topic" dashboard shows "subscription" stats 🤔